### DISCUSSION TITLE
Support for Database Backend?

### DISCUSSION THREAD

**Discussion by bisoogo** [Textualize/textual#3388] [Q&A] [answered]
Hello,
Is there support for a database backend?

**Comment by davep** [comment, marked as answer]
Textual has no native support for any database backend, but plenty of people are using various database libraries to build applications where they use Textual to build the user interface.

**Comment by bradlarsen** [comment]
See also: https://github.com/Textualize/textual/discussions/2780

I just found this package, which I am going to try: https://pypi.org/project/textual-fastdatatable/

"A performance-focused reimplementation of Textual's DataTable widget, with a pluggable data storage backend."
